test/e2e/major_upgrade_v2.go

```diff
@@ -12,28 +12,40 @@ import (
 	v1 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v1"
 	v2 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v2"
 	"github.com/celestiaorg/celestia-app/v3/test/e2e/testnet"
+	"github.com/celestiaorg/knuu/pkg/knuu"
 	"github.com/tendermint/tendermint/rpc/client/http"
 )
 
 func MajorUpgradeToV2(logger *log.Logger) error {
 	var (
+		testName      = "MajorUpgradeToV2"
 		numNodes      = 4
 		upgradeHeight = int64(10)
 	)
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	identifier := fmt.Sprintf("%s_%s", testName, time.Now().Format(timeFormat))
+	kn, err := knuu.New(ctx, knuu.Options{
+		Scope:        identifier,
+		ProxyEnabled: true,
+	})
+	testnet.NoError("failed to initialize Knuu", err)
+
+	kn.HandleStopSignal(ctx)
+	logger.Printf("Knuu initialized with scope %s", kn.Scope)
```

Comment:
[nit] same nit about moving this immediately after knuu initialization.

Reply:
it is after knuu initialization.  `knuu.New()` does the initialization and if it does not fail (error check), then we handle the stop signal.

or you mean removing that empty line?

Reply:
I mean moving the log statement.

```diff
	kn, err := knuu.New(ctx, knuu.Options{
		Scope:        scope,
		ProxyEnabled: true,
	})
	testnet.NoError("failed to initialize Knuu", err)
+	logger.Printf("Knuu initialized with scope %s", kn.Scope)

	kn.HandleStopSignal(ctx)
-	logger.Printf("Knuu initialized with scope %s", kn.Scope)
```